settings.gradle

```diff
@@ -1,6 +1,6 @@
 apply from: 'buildSdk.gradle'
 
-include ':app-sample', ':modules-common', ':modules-browsemap', ':uikit-common', ':uikit-common-sdk', ':uikit-compass', ':uikit-compass-viewmodel',
+include ':app-sample', ':modules-common', ':modules-browsemap', ':uikit-common', ':uikit-common-sdk', ':uikit-compass', ':uikit-compass-viewmodel', ':injector', ':injector-viewmodel',
```

Comment:
Nechtěli bychom prosím zachovat nějakou tu logickou strukturu, o kterou jsem se tu předtím pokoušel? :) každý má momentálně nějaký prefix + parent folder a tím je zajištěna přehledná struktura. Když jsem si doma dnes checkoutl repo, byl jsem nadšený, jak je to na první pohled hned přehledné a pěkně seřazené :-D 
![image](https://user-images.githubusercontent.com/4901251/49083119-00396d80-f24c-11e8-8180-c1147868041c.png)
Nešlo by tedy pro tento nový "injector" modul a jeho VM vymyslet nějaký nový parent případně zařadit mezi stávající, např:
uikit-common-injector (aby mohli používat jak hlavní moduly, tak uikt moduly)
? :)

Reply:
no on je prave nezavisly aj od `modules` aj od `uikit`. Preto som dal ako parent `injector` a vm je v tomto parentovi. Jasne ze mozem dat este jednu uroven, ale nie je to ani kit ani moduly. Tak neviem co by som tam dal? `tools`?